packages/charts/src/charts/pie/pie-chart.component.tsx

```diff
@@ -76,11 +89,15 @@ export const PieChart: FC<Props> = ({
     keys,
     disabledLabels,
     labelsPosition,
-    stackTreshold,
     dimension: svgDimensions,
     colors: theme.colors,
+    treshold,
   });
 
+  useEffect(() => {
+    onChange(dataKeys);
```

Comment:
As we discussed - the handler should notify parent about stacked data series. Please use already defined interface:
```
{ selector: DataSelector; color: string }[];
```